packages/dx-react-chart/src/plugins/selection-state.jsx

```diff
@@ -21,10 +22,10 @@ export class SelectionState extends React.PureComponent {
 SelectionState.propTypes = {
   selection: PropTypes.arrayOf(PropTypes.shape({
     series: PropTypes.string.isRequired,
-    point: PropTypes.number,
+    point: PropTypes.number.isRequired,
```

Comment:
What about if I want select/hover only series, without point?

Reply:
The documentation should be fix, too

Reply:
It is in another [branch](https://github.com/DmitryBogomolov/devextreme-reactive/tree/update-hover-docs).